employee_manual/travelling.md

```diff
@@ -0,0 +1,29 @@
+# Travelling
+
+## Commuting to your main office
+Commuting does not count as work time.
+
+However, if your commute distance is long enough you might be eligible for "Deduction for transport to work" (Befordringsfradrag). More information is available at [skat.dk](https://skat.dk) (use the search feature).
+
+## Travelling between offices
+You are NOT expected to work overtime to cover time spent travelling, and Abtion will cover your expenses (get in touch with your PM for further details).
```

Comment:
I would remove the responsibility from the PM and place it with Dafna.

Reply:
I would prefer to avoid names of current employees in the guidelines.
Which role should I write here? Office manager seems a bit unrelated to travelling expences.

Reply:
I agree about avoiding names to the extend it is possible, but it is Dafna who takes care of it for the time being (not even "Office manager", because Alberte doesn't take care of those things).